### DISCUSSION TITLE
Backend prompts

### DISCUSSION THREAD

**Discussion by lDrakohh** [ispp-2324-ocial/KB#3] [Prompts IAS]
Insert here prompts use to generate backend content

**Comment by Domi-ATLAS** [comment]
Hemos usado el siguiente prompt, Carlos Varela y Eduardo Pizarro, para saber si un id se genera automaticamente al crear una clase: https://chat.openai.com/share/2c0d1ebb-57a8-4148-853e-2223cb62aa27

**Comment by Domi-ATLAS** [comment]
Hemos usado el siguiente prompt, Carlos Varela y Eduardo Pizarro, para saber por que el test daba error: https://chat.openai.com/share/2bedeb20-88ba-4ddf-80d2-7d919d05187d

**Comment by cvsoult** [comment]
Hemos usado el siguiente prompt, Carlos Varela y Eduardo Pizarro, para obtener dos instancias de ejemplo como referencia para el EventTestCase:
https://chat.openai.com/share/8dec3d9b-3c06-4514-accd-0bd2f2a04f58

**Comment by cvsoult** [comment]
Hemos usado el siguiente prompt, Carlos Varela y Raúl Montalbán, para obtener una validación de la letra del DNI con RegexValidator:
https://chat.openai.com/share/3db15016-0d99-4c4a-a218-bc171f3a1135

**Comment by iniruimar** [comment]
Hemos usado el siguiente prompt, Jose Carlos Ortiz e Iñigo Ruiz, para solucionar unos errores y para ayudarnos a entender como hacer una validación: https://chat.openai.com/share/f923705f-8d26-45c7-be29-1c556651290e
